src/views/Login/index.tsx

```diff
@@ -21,6 +26,18 @@ const Login = () => {
     setIsLocalAuth(true);
   };
 
+  const onSuccess = (tokens: any) => {
+    oktaAuth.handleLoginRedirect(tokens);
+  };
+
+  useEffect(() => {
```

Comment:
This actually isn't needed. I thought it was a good touch to redirect a user to the come page if they're already authenticated (so they don't sign in again).